src/IssueViz.Security.UnitTests/HotspotsList/HotspotTableEntryTests.cs

```diff
@@ -201,5 +222,25 @@ private static object GetValue(IAnalysisIssue issue, string column)
 
             return value;
         }
+
+        private IAnalysisIssueVisualization CreateIssueVizWithSpan()
```

Comment:
Copied from IssuesSnapshotTests_GetValue

Reply:
I assumed "Copied from X" meant the whole method was copied so I was a bit confused that that isn't the case - looks like it's part of the constructor.

I think's worth changing the method name to emphasise that the returned object is shared and the individual properties matter e.g. `CreateWellKnownIssueVizAndSpan` - it's more of a warning to future maintainers that changing anything in the function might impact multiple tests.

In general I try to avoid sharing a complex object over multiple tests where the tests check specific individual values; it breaks test isolation and makes tests harder to read. If I do have to shared objects between tests then I try to make sure the tests don't depend on its properties i.e. the test just needs a valid snapshot but doesn't care about the content.